ex11/part3/src/randomLabelCheck.js

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from 'react';
+
+export default function randomLabelCheck(prop) {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [textb] = useState(Math.floor(Math.random() * 1000));
```

Comment:
If there is no setter then there is no reason to use state, just use a regular variable.